pkg/config/logger.go

```diff
@@ -18,11 +18,14 @@ func NewLogger() *zap.SugaredLogger {
 		log.Fatal(err)
 	}
 
-	logFilePath := filepath.Join(logDir, "massastation.log")
+	logFilePath := filepath.Join(logDir, "logs", "massastation.log")
+	logDirPath := filepath.Dir(logFilePath)
```

Comment:
I feel it's more readable like this:
```suggestion
	logDirPath := filepath.Join(logDir, "logs")
	logFilePath := filepath.Join(logDirPath, "massastation.log")
```

Reply:
done